scripts/env-setup.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+################################################
+# Generates and sets environment variables for broker
+#
+################################################
+
+# Set the network in the .env file based on user input
+echo "Enter the network: 1 for MainNet, 2 for TestNet"
```

Comment:
hmm, I think I'd prefer letters (e.g. `m` for mainnet, `t` for testnet) over numbers. Why did you choose to do it this way?

Also, might be nice to display them in a list, e.g.

```
Enter the network:
m - MainNet
t - TestNet
```